### ISSUE TITLE
add pastas.timeseries.TimeSeries object to a PastaStore using PastaStore.add_oseries()

### ISSUE THREAD

**Issue by OnnoEbbens** [pastas/pastastore#34]
I think it would be nice to modify `PastaStore.add_oseries()` in such a way that the `series` argument can be a `pastas.timeseries.TimeSeries` object. Now only pandas `DataFrame` and `Series` are supported. It can save you some code, for example when adding a Dino csv file to the store.

Now:
```
series = ps.read_dino(fname)
store.add_oseries(series.series_original, name=series.name, metadata=series.metadata)
```

Proposed change:
```
series = ps.read_dino(fname)
store.add_oseries(series)
```

What are your thoughts on this?

**Comment by dbrakenhoff** [comment]
Good idea! 

Only problem I can see is that it might become confusing when the user is not getting back a pastas.TimeSeries object when retrieving data. But making it easy to add pastas.TimeSeries is probably worth that potential (minor) confusion.

**Comment by dbrakenhoff** [comment]
The name still has to be passed individually, but timeseries and metadata can be taken from the pastas.TimeSeries object.

```python
series = ps.read_dino(fname)
store.add_oseries(series, series.name)
```

Closed by #40
